### PR TITLE
Use template instead of stringbuilder

### LINKED ISSUE #5: Use go template instead of string builder for GitHub issues
Hi @alingse ,
Nice work on go-linter-runner.
I noticed that you've used a string builder to create your GitHub issues comments.

Have you considered using a golang template like https://pkg.go.dev/html/template to produce them? 
I believe that would be easier to review and modify them .

### PR DESCRIPTION
**Pull request by mmorel-35**
Closes #5

The lines could be more templated but this can be done in a second time. This provides a template to create the comment in the issue concerning the execution of a linter on a repository

### CODE REVIEW

File: runner/run/execute.go [hunk 1]
```diff
@@ -184,22 +186,33 @@ func excludeLine(excludes []string, line string) bool {
...
+		RepositoryURL: cfg.Repo,
+		
+	}
+	for _, line := range outputs {
 		text := buildIssueCommentLine(cfg, line)
-		s.WriteString(fmt.Sprintf("%d. %s\n", i+1, text))
+		data.Lines = append(data.Lines, text)
+	}
+	var tpl bytes.Buffer
+	tmpl := template.Must(template.New("issue_comment").Parse(issueCommentTemplate))
```
Comment (ccoVeille) [inline runner/run/execute.go:211]: I'm unsure about the Must here

I know it's open to debate.

```suggestion
	tmpl, err := template.New("issue_comment").Parse(issueCommentTemplate)

  if err != nil {
       return "", err
  }
```

**Review by ccoVeille — APPROVED** [review]

File: runner/run/execute.go (hunk 1 above)
Comment (ccoVeille) [reply runner/run/execute.go]: Thanks for applying my suggestion, but I'm afraid the code I suggested via my phone needs to be gofmt-ed

**Comment by alingse** [comment]
thx so much. @mmorel-35 @ccoVeille

**Comment by ccoVeille** [comment]
Thanks for the invitation on the org

**Pull request merged**